src/page/sinfo.rs

```diff
@@ -83,26 +94,41 @@ mod test {
 
     #[test]
     fn display() {
-        assert_eq!(format!("{}", SecInfo::tcs()), "T");
-        assert_eq!(format!("{}", SecInfo::reg(Flags::READ)), "R");
-        assert_eq!(format!("{}", SecInfo::reg(Flags::WRITE)), "W");
-        assert_eq!(format!("{}", SecInfo::reg(Flags::EXECUTE)), "X");
+        assert_eq!(format!("{}", SecInfo::from(Class::Tcs)), "T");
+        assert_eq!(format!("{}", SecInfo::from(Class::Regular)), "R");
+        assert_eq!(
+            format!("{}", SecInfo::new(Class::Regular, Flags::WRITE)),
+            "W"
+        );
+        assert_eq!(
+            format!("{}", SecInfo::new(Class::Regular, Flags::EXECUTE)),
+            "X"
+        );
```

Comment:
Notice how `SecInfo::new(...)` makes this line too long. I wonder if we should turn `SecInfo::new(...)` into `Class::info(self, flags: impl Into<Option<Flags>>)` instead...

Reply:
Then we could do:

```rust
Class::Regular.info(None)
```

or 

```rust
Class::Regular.info(Flags::READ | Flags::WRITE);
```